kernel/src/table_features/mod.rs

```diff
@@ -109,17 +115,25 @@ pub enum WriterFeatures {
     /// vacuumProtocolCheck ReaderWriter feature ensures consistent application of reader and writer
     /// protocol checks during VACUUM operations
     VacuumProtocolCheck,
+    #[serde(untagged)]
```

Comment:
no `#[strum(to_string = "{0}")]`?

Reply:
bump on this

Reply:
I realized it was missing here, thanks for pointing that out